File: src/main/java/hdzi/editstarters/initializr/InitializrVersion.java
package hdzi.editstarters.initializr;

import com.google.gson.annotations.SerializedName;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class InitializrVersion {
    @Getter
    @Setter
    public static class Value {
        private String id;
        private String name;
    }

    @SerializedName("default")
    private String _default;
    private List<Value> values;
}

File: src/main/kotlin/hdzi/editstarters/bean/initializr/InitializrVersion.kt
package hdzi.editstarters.bean.initializr

class InitializrVersion {
    var default: String? = null
    var values: List<Value>? = null

    class Value {
        var id: String? = null
        var name: String? = null
        override fun toString(): String {
            return name!!
        }
    }
}

